plugins/openchoreo-api/src/types/discovery.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * This is a copy of the DiscoveryApi, to avoid importing core-plugin-api.
```

Comment:
Why do we need to do this?

Reply:
fixed with https://github.com/openchoreo/backstage-plugins/pull/5/commits/476ece418ba0602dbe17f0f3bc9daf3c58398708